streams/src/main/java/org/apache/kafka/streams/kstream/internals/TimeWindowedKStreamImpl.java

```diff
@@ -135,6 +135,7 @@
         Objects.requireNonNull(materialized, "materialized can't be null");
 
         final MaterializedInternal<K, V, WindowStore<Bytes, byte[]>> materializedInternal = new MaterializedInternal<>(materialized);
+        final Aggregator<K, V, V> reduceAggregator = aggregatorForReducer(reducer);
```

Comment:
Nit: I would inline this method. It's a single liner and only used once.

Reply:
I have inlined it, good catch.